doc/manual/noteworthy-differences.rst

```diff
@@ -100,6 +100,14 @@ some noteworthy differences that may trip up Julia users accustomed to MATLAB:
   unlike MATLAB's ``class``\ es. Instead, use a :obj:`Dict`.
 - In Julia each module has its own global scope/namespace, whereas in
   Matlab there is just one global scope.
+- In MATLAB, an idiomatic way to remove unwanted values is to use logical
+  indexing, like in the expression ``x(x>3)``. Julia provides a :func:`filter!`
+  function for this purpose, so that one can write
+  ``filter!(z->z>3, x)`` rather than the transliteration ``x[x.>3]``,
+- The analogue of extracting (or "dereferencing") all elements of a
+  one-dimensional cell array, e.g. in ``vertcat(A{:})`` in MATLAB, is written
```

Comment:
I don't think this is restricted to 1D IIRC. Been nearly 2.5 years since I wrote any Matlab, but I did use this pattern a lot.